diff --git a/travis.sh b/travis.sh
index c05e7c4..9fd5032 100755
--- a/travis.sh
+++ b/travis.sh
@@ -8,7 +8,7 @@ function travis_time_start {
     TRAVIS_TIME_ID=$(cat /dev/urandom | tr -dc 'a-z0-9' | fold -w 8 | head -n 1)
     TRAVIS_FOLD_NAME=$1
     echo -e "\e[0Ktraivs_fold:start:$TRAVIS_FOLD_NAME"
-    echo -e "\e[0Ktraivs_time:start:$TRAVIS_TIME_ID"
+    echo -e "\e[0Ktraivs_time:start:$TRAVIS_TIME_ID\e[44m\n\e[34m>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\e[0m"
     set -x
 }
 function travis_time_end {
@@ -17,7 +17,7 @@ function travis_time_end {
     TRAVIS_END_TIME=$(date +%s%N)
     TIME_ELAPSED_SECONDS=$(( ($TRAVIS_END_TIME - $TRAVIS_START_TIME)/1000000000 ))
     echo -e "traivs_time:end:$TRAVIS_TIME_ID:start=$TRAVIS_START_TIME,finish=$TRAVIS_END_TIME,duration=$(($TRAVIS_END_TIME - $TRAVIS_START_TIME))\n\e[0K"
-    echo -e "traivs_fold:end:$TRAVIS_FOLD_NAME"
+    echo -e "traivs_fold:end:$TRAVIS_FOLD_NAME\e[$(expr 10 + ${_COLOR})m\n\e[${_COLOR}m<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<\e[0m"
     echo -e "\e[0K\e[${_COLOR}mFunction $TRAVIS_FOLD_NAME takes $(( $TIME_ELAPSED_SECONDS / 60 )) min $(( $TIME_ELAPSED_SECONDS % 60 )) sec\e[0m"
     set -x
 }
